interpreter/variable/shared.go

```diff
@@ -294,9 +294,7 @@ func SetWafVariables(ctx *context.Context, name, operator string, val value.Valu
 			return true, errors.WithStack(err)
 		}
 		return true, nil
-	case WAF_FAILURES,
-		"waf.php_injection_score",
-		"waf.rce_score":
+	case WAF_FAILURES, WAF_PHP_INJECTION_SCORE, WAF_RCE_SCORE:
```

Comment:
According to [waf.php_injection_score](https://github.com/ysugimoto/falco/pull/245/files#diff-8ef80a0b3818237e075db8a4ed57ac59ec7743d0f04f1e1cc375a7f7ff170bb0R2174) and [waf.rce_score](https://github.com/ysugimoto/falco/pull/245/files#diff-8ef80a0b3818237e075db8a4ed57ac59ec7743d0f04f1e1cc375a7f7ff170bb0R2180) ,  `waf.php_injection_score` and `waf.rce_score` could be set.

Reply:
Thanks for noticing that! I had fixed the typo in `predefined.yml` but forgot to actually make the related changes in the interpreter. Done.